### ISSUE TITLE
Demo after Install does not work.

### ISSUE THREAD

**Issue by jersol** [chatziko/location-guard#10]
Demo after Install does not work on Mozilla Firefox Ver. 41.0 (Latest)  under Windows 7.

**Comment by chatziko** [comment]
Thanks for the report, I can reproduce the problem.

In other pages (apart from the demo), eg http://html5demos.com/geo, the extension works, right?

**Comment by chatziko** [comment]
Turns out this due a [bug in Firefox 41](https://bugzilla.mozilla.org/show_bug.cgi?id=1208609). Thankfull it seems to be fixed in Firefox 42 (currently in the beta channel).

I uploaded a new version of Location Guard ([1.2.1](https://addons.mozilla.org/en-US/firefox/addon/location-guard/versions/?page=1#version-1.2.1), waiting for approval). This version disables the demo page under Firefox 41 desktop (android is ok) to avoid a broken user experience. Note that Location Guard still works fine, only the demo page has issues.

I'll keep this bug open until Firefox 42 is out.
